sub-logs/aaron.json

```diff
@@ -0,0 +1,17 @@
+{
+    "repo": "https://github.com/Adriichu/is-a-good-dev",
+  
+    "owner": {
+      "username": "Adriichuuu",
+      "email": "aaron@proton.me"
+    },
+  
+    "target": {
+      "record-type": {
```

Comment:
Please change `record-type` to `CNAME`, as this looks like the type of record you are using.

Reply:
Sorry, I completely forgot about that.